src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -50,8 +50,9 @@
     </PropertyGroup>
 
     <PropertyGroup>
+      <MaxSupportedLangVersion Condition="'$(MaxSupportedLangVersion)' == ''">7.3</MaxSupportedLangVersion>
```

Comment:
This still seems wrong. This is explicitly only true when targeting older versions of netcore or framework.

Reply:
Ah, yeah, you're right, I need to lift the condition out.